api/src/peoplemeasurement/migrations/0008_cmsa_15min_view_v2_adjusted_20200525_2120.py

```diff
@@ -0,0 +1,138 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('peoplemeasurement', '0007_druktemonitor_hourly_view_20200508_1506'),
+    ]
+
+    _VIEW_NAME = "cmsa_15min_view_v2"
+
+    sql = f"""
+CREATE VIEW {_VIEW_NAME} AS
```

Comment:
Als je deze sql naar een aparte file gaat, dan is het makkelijker daar changes in te tracken.

Reply:
Ga ik bij een volgende view migratie doen